src/test/java/org/gridsuite/geodata/server/GeoDataControllerTest.java

```diff
@@ -87,6 +88,8 @@ public void test() throws Exception {
         Network testNetwork = EurostagTutorialExample1Factory.create();
         testNetwork.getVariantManager().cloneVariant(VariantManagerConstants.INITIAL_VARIANT_ID, VARIANT_ID);
         given(service.getNetwork(networkUuid)).willReturn(testNetwork);
+        given(service.getNetwork(networkUuid, PreloadingStrategy.NONE)).willReturn(testNetwork);
+        given(service.getNetwork(networkUuid, PreloadingStrategy.COLLECTION)).willReturn(testNetwork);
```

Comment:
We return the same `testNetwork` independently of the `PreloadingStrategy`, how do we test that it has an effect then ?